bokehjs/src/lib/models/widgets/tables/row_aggregators.ts

```diff
@@ -0,0 +1,96 @@
+const { Aggregators: { Avg, Min, Max, Sum } } = require('slickgrid/slick.dataview')
+
+import * as p from 'core/properties'
+import { Model } from 'model'
+
+export type GroupTotals = {
+  avg: { [field_: string]: number }
+  min: { [field_: string]: number }
+  max: { [field_: string]: number }
+  sum: { [field_: string]: number }
+}
+
+export namespace RowAggregator {
+  export type Attrs = p.AttrsOf<Props>
+
+  export type Props = Model.Props & {
+    field_: p.Property<string>
```

Comment:
Why use training underscore? I suppose this does conflict with something else?

Reply:
The underscore is expected by the underlying SlickGrid implementations.